core/Plugin/Menu.php

```diff
@@ -30,16 +30,6 @@
  */
 class Menu
 {
-    protected $module = '';
```

Comment:
It is there for performance if I remember correctly. We will do it only once instead of many times